test/integration/discovery.v2.test.js

```diff
@@ -222,6 +241,30 @@ describe('discovery v2 integration @slow', () => {
       expect(result).toBeDefined();
       done();
     });
+
+    test('analyzeDocument', async done => {
+      if (!documentId) {
+        return done();
+      }
+      const params = {
+        projectId,
+        collectionId,
+        file: fs.createReadStream(path.join(__dirname, '../resources/sampleWord.docx')),
+        filename: 'sampleWord.docx',
+      };
+
+      let res;
+      try {
+        res = await discovery.analyzeDocument(params);
+      } catch (err) {
+        return done(err);
+      }
+
+      expect(res).toBeDefined();
+      const { result } = res || {};
+      expect(result).toBeDefined();
+      done();
+    });
```

Comment:
Looking at the discovery code, it looks like way more methods than just these two were added. Maybe I'm missing something but if not, we should add integration tests for all of the new methods as well

Reply:
That is strange, I don't know how I missed all those new methods... I will make sure to write tests for them

Reply:
I think that a lot of the methods I saw that looked like they were added were old ones that just got moved (so they looked like additions). So I misinterpreted. You can check if there are already tests for some of these operations, just search in the code. No need to duplicate any testing